src/goto-programs/goto_contractor.cpp

```diff
@@ -967,47 +989,42 @@ expr2tc interval_analysis_ibex_contractor::result_of_outer(expr2tc exp)
         continue;
 
       //if there is a lower bound,
-      if (!isinf(var.second.getInterval().lb()))
+      if (isnormal(var.second.getInterval().lb()))
```

Comment:
Would `isfinite()` work here, too?

Reply:
I am not sure but does isfinite() include NaN? if not I'll change it.

Reply:
According to [documentation](https://linux.die.net/man/3/isfinite), isfinite() will be true when the argument is neither infinite nor NaN.